### PR TITLE
DATA-38 - Applying some initial changes.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR applies some changes in order to run the edx-analytics-exporter pipeline.

Changes:
- Removing support for encrypted files.
- Removing user and password for mongoexport command call.
- Adding task to get the courseware_studentmodule data.

## Reviewers:

- [ ] @andrey-canon 
- [ ] @diegomillan

### CODE REVIEW

File: exporter/tasks.py [hunk 1]
```diff
@@ -152,18 +152,21 @@ def get_values(cls, **kwargs):
 
 
 class MongoTask(Task):
+    """
+    The mongo task to export the Mongo DB dump.
+
+    Initially the mongoexport command take the --username and --password parameters,
```
Comment (andrey-canon) [inline exporter/tasks.py]: grammatical error, it's `takes`

File: exporter/tasks.py [hunk 2]
```diff
@@ -273,16 +275,20 @@ def run(cls, filename, dry_run, **kwargs):
...
-                # Define retries here, to recover from temporary outages when calling S3 to find files.
-                local_kwargs = dict(**kwargs)
-                local_kwargs['max_tries'] = MAX_TRIES_FOR_MARKER_FILE_CHECK
-                execute_shell(marker_command, **local_kwargs)
-            except subprocess.CalledProcessError:
-                error_message = 'Unable to find success marker for export {0}'.format(s3_marker_filename)
-                log.error(error_message)
-                raise FatalTaskError(error_message)
+            # This code block was commented since, the first time we run this,
+            # we don't have data related to the previous run, so
```
Comment (andrey-canon) [inline exporter/tasks.py:279]: grammatical error, it's 'related data'

File: exporter/tasks.py [hunk 3]
```diff
@@ -273,16 +275,20 @@ def run(cls, filename, dry_run, **kwargs):
...
+            # this always raise a FatalTaskError exception, and we don't want that.
```
Comment (andrey-canon) [inline exporter/tasks.py]: grammatical error, 'raises'

File: exporter/tasks.py [hunk 4]
```diff
@@ -322,9 +328,14 @@ class UserIDMapTask(CourseTask, SQLTask):
     """
 
 
-class StudentModuleTask(CourseTask, CopyS3FileTask):
+class StudentModuleTask(CourseTask, SQLTask, CopyS3FileTask):
```
Comment (andrey-canon) [inline exporter/tasks.py:331]: How does it work ?

**Review by andrey-canon — CHANGES_REQUESTED** [review]

File: exporter/tasks.py (hunk 1 above)
Comment (Squirrel18) [reply exporter/tasks.py]: Addressed.

File: exporter/tasks.py (hunk 2 above)
Comment (Squirrel18) [reply exporter/tasks.py:279]: No, it's fine.

File: exporter/tasks.py (hunk 3 above)
Comment (Squirrel18) [reply exporter/tasks.py]: Addresed.

File: exporter/tasks.py (hunk 4 above)
Comment (Squirrel18) [reply exporter/tasks.py:331]: Check SQLTask class...

**Comment by Squirrel18** [comment]
Check again. @andrey-canon

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**